tests/test_herbstclient.py

```diff
@@ -85,7 +85,7 @@ def test_herbstclient_wait(hlwm, num_hooks_sent, num_hooks_recv):
         hlwm.call('emit_hook nonmatch nonarg')
         hlwm.call('emit_hook matcher somearg')
 
-    proc.wait(3)
+    proc.wait(5)
```

Comment:
Should we extract the timeout to a variable used in both tests?

Reply:
not sure whether it's an improvement

Reply:
It's not that important to me, either ;)